eslint/babel-eslint-parser/test/integration.js

```diff
@@ -1,16 +1,15 @@
 "use strict";
 
-const assert = require("assert");
 const eslint = require("eslint");
 const fs = require("fs");
 const path = require("path");
 
-const parser = require("../..");
+const parser = require("../lib");
```

Comment:
Nit: it suffices to use `..` and resolve using metadata (e.g. `main`) in `package.json`.